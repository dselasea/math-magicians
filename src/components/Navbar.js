import React from 'react';
import { Link } from 'react-router-dom';

const Navbar = () => (
  <nav>
    <Link to="/">Home</Link>
    <Link to="/calculator">Calculator</Link>
    <Link to="/quote">Quotes</Link>
  </nav>
);

export default Navbar;
