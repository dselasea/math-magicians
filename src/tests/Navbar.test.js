import renderer from 'react-test-renderer';
import Navbar from '../components/Navbar';

test('Test Navbar Component', () => {
  const navbarComponent = renderer.create(<Navbar />);
  const navbar = navbarComponent.toJSON();
  expect(navbar).toMatchSnapshot();
});
