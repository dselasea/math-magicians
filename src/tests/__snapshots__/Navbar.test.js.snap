// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Navbar Component 1`] = `
<div
  className="nav-container"
>
  <nav
    className="nav"
  >
    <div>
      <h1>
        Math Magicians
      </h1>
    </div>
    <div
      className="links"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
      <a
        href="/quote"
        onClick={[Function]}
      >
        Quotes
      </a>
    </div>
  </nav>
</div>
`;
